test/example_data.dart

```diff
@@ -180,6 +181,13 @@ ZulipStream stream({
 }
 const _stream = stream;
 
+Topic topic({
+  int? maxId,
+  String? name,
+}) => Topic(
+  maxId: maxId ?? 123,
+  name: name ?? 'Test Topic #$maxId');
```

Comment:
If `maxId` and `name` are both omitted, won't this say 'Test Topic #null' or something?